developers/addon/index.md

```diff
@@ -1,14 +1,29 @@
 ---
 layout: developersguide
-title: Bindings
+title: Add-ons
 ---
 
-# Developing a Binding
+# Developing an add-on
 
 {:.no_toc}
 
-A binding is an extension to openHAB that integrates an external system like a software service or a hardware device.
-The external system is represented as a set of _Things_ and sometimes _Bridges_ with _Channels_.
+An add-on is an extension to openHAB that integrates external components like software services or hardware devices.
+Depending on their use-case they have different types:
+
+- `automation` for add-ons that provide scripting languages or rule templates
+- `binding` for controlling physical devices or software systems
+- `misc` for other integrations like Apple's HomeKit or the openHAB cloud
+- `persistence` for services that can be used to store and read historical data
+- `transform` for add-on that provide a way to manipulate values
+- `ui` for bundles that provide UIs or UI-additions (like widgets)
+- `voice` for add-ons that provide voice services
+
+## Structure of an Add-on
+
+Every add-on needs to define a `addon.xml` file.
+Find more information in the respective [Add-on XML reference](addon-xml.html).
```

Comment:
This url will result in a 404.
Probably `addon.html` could be meant, since the file was added, but i am not sure.

```suggestion
Find more information in the respective [Add-on XML reference](addon.html).
```